server/src/main/java/com/paypal/selion/utils/ServletHelper.java

```diff
@@ -37,6 +37,9 @@
  * 
  */
```

Comment:
The javadoc talks about commonly repeated HTML code .. which is no longer the case. :)